copter/source/docs/terrain-following.rst

```diff
@@ -13,7 +13,7 @@ See :ref:`common-understanding-altitude` for altitude definitions.
 
 .. note::
 
-   :ref:`Loiter <loiter-mode>`, :ref:`PosHold <poshold-mode>` and :ref:`AltHold <altholdmode>` modes also support low altitude terrain following, called Surface Tracking.  Please refer to the :ref:`Surface Tracking <terrain-following-manual-modes>` wiki page.
+   :ref:`Loiter <loiter-mode>`, :ref:`PosHold <poshold-mode>` and :ref:`AltHold <altholdmode>` modes also support low altitude terrain following, called Surface Tracking. Surface tracking uses a rangefinder(s) oriented either pointing down or up or both and which surface is tracked (floor or ceiling) is controlled by the :ref:`SURFTRAK_MODE<SURFTRAK_MODE>` parameter.  Please refer to the :ref:`Surface Tracking <terrain-following-manual-modes>` wiki page.
```

Comment:
I think we should keep this line as it was.  No need to go into the details of up/down here.